### PR TITLE
Add grafana metrics dashboard

### PR DESCRIPTION
**Pull request by sefacan**

### CODE REVIEW

**Review by erayarslan — APPROVED** [review]

File: dashboards/metrics.json [hunk 1]
```diff
@@ -0,0 +1,2060 @@
+{
```
Comment (Abdulsametileri) [inline dashboards/metrics.json:1]: Could you provide screen shoot?

**Review by Abdulsametileri — CHANGES_REQUESTED** [review]

**Comment by Abdulsametileri** [comment]
This dashboard is not applicable for open source (it includes our logics) 
we need to prepare generic dashboard like https://github.com/Trendyol/kafka-konsumer/tree/v2/examples/with-grafana
could you change like that

File: dashboards/metrics.json (hunk 1 above)
Comment (sefacan) [reply dashboards/metrics.json:1]: I have added.

**Comment by sefacan** [comment]
> This dashboard is not applicable for open source (it includes our logics) we need to prepare generic dashboard like https://github.com/Trendyol/kafka-konsumer/tree/v2/examples/with-grafana could you change like that

Ok I will provide

**Comment by sefacan** [comment]
> This dashboard is not applicable for open source (it includes our logics) we need to prepare generic dashboard like https://github.com/Trendyol/kafka-konsumer/tree/v2/examples/with-grafana could you change like that

Hello again, I have added quick docker-compose example for grafana.

**Comment by Abdulsametileri** [comment]
Sorry for my late response, lgtm, could you solve the conflict?

**Pull request closed without merging**